test/acceptance/tests/mesh-gateway/mesh_gateway_test.go

```diff
@@ -3,10 +3,13 @@ package meshgateway
 import (
 	"fmt"
 	"testing"
+	"time"
 
 	"github.com/hashicorp/consul-helm/test/acceptance/framework"
 	"github.com/hashicorp/consul-helm/test/acceptance/helpers"
 	"github.com/hashicorp/consul/api"
+	"github.com/hashicorp/serf/serf"
+	"github.com/hashicorp/serf/testutil/retry"
```

Comment:
Using serf's retry here

Reply:
Oh, you're right. I was surprised when I brought in the serf const why go.mod wasn't modified! Let me correct!